scripts/apt-install-defaults-plus-args

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Can we namespace these based on the base container? E.g., for ubuntu it should either be in one of these places I think:

```
ubuntu/
   scripts/<scripts for ubuntu>
```
or 
```
ubuntu/
scripts/
   ubuntu/<scripts for ubuntu>
```
What do you think?

Reply:
Sure, I don't mind.  The spack-setup one would probably be usable/useful for all of them though if the plan is to use spack from all the bases.

Reply:
Whoops, didn't realize this one was still open. I'm going to close this one so we can work on the one using the upstream branch.